backend/entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+echo "Waiting for postgres..."
+
+while ! nc -z 'postgres' 5432; do
+  echo "Zzz"
+  sleep 0.5
+done
+
+echo "PostgreSQL started"
+
+python manage.py flush --no-input
+python manage.py migrate
+
+
+if [ -z "$(python manage.py shell -c 'from django.contrib.auth.models import User; print(User.objects.filter(username="your_superuser_username").exists())')" ]; then
```

Comment:
`your_superuser_username` is a literal string. Make use of a real variable or an environment variable